src/lang/Messages.properties

```diff
@@ -1773,8 +1773,9 @@ spider.custom.button.reset	= Reset
 spider.custom.button.scan	= Start Scan
 spider.custom.label.adv		= Show Advanced options
 spider.custom.label.context	= Context:
-spider.custom.label.maxChildren		= Maximum children to crawl:
-spider.custom.label.maxDepth		= Maximum depth to crawl:
+spider.custom.label.maxChildren		= Maximum children to crawl (0 is unlimited):
+spider.custom.label.maxDepth		= Maximum depth to crawl (0 is unlimited):
```

Comment:
In the UI it does not allow to select 0, the minimum is 1 (the actual check expects a value greater than 0).